### PR TITLE
Adding add storage test case for longevity

### PR DESCRIPTION
**Pull request by lsrinivas-pure**
**What this PR does / why we need it**:

Adding longevity tests for adding new storage node and storageless nodes for OCP clusters

**Which issue(s) this PR fixes** (optional)
Closes #PTX-18926

**Special notes for your reviewer**:
Aetos link : https://aetos.pwx.purestorage.com/resultSet/testSetID/306616

### CODE REVIEW

File: tests/basic/longevity_test.go [hunk 1]
```diff
@@ -1355,7 +1357,7 @@ func populateIntervals() {
 	triggerInterval[VolumeCreatePxRestart][2] = 9 * baseInterval
 	triggerInterval[VolumeCreatePxRestart][1] = 10 * baseInterval
 
-	baseInterval = 300 * time.Minute
+	baseInterval = 2 * time.Minute
```
Comment (vprabhakar-px) [inline tests/basic/longevity_test.go:1360]: revert base interval of 2 min to original value

File: tests/common.go [hunk 2]
```diff
@@ -2074,12 +2073,18 @@ func CollectSupport() {
 	context("generating support bundle...", func() {
 		log.InfoD("generating support bundle...")
 		skipStr := os.Getenv(envSkipDiagCollection)
+		skipSystemCheck := false
+
 		if skipStr != "" {
 			if skip, err := strconv.ParseBool(skipStr); err == nil && skip {
-				log.Infof("skipping diag collection because env var %s=%s", envSkipDiagCollection, skipStr)
```
Comment (vprabhakar-px) [inline tests/common.go:2079]: can we log the error

Comment (lsrinivas-pure) [reply tests/common.go:2118]: moved this log to line 2125

File: tests/basic/longevity_test.go (hunk 1 above)
Comment (lsrinivas-pure) [reply tests/basic/longevity_test.go]: fixed

**Review by aravindranath14 — APPROVED** [review]
LGTM

**Pull request merged**